### ISSUE TITLE
Compile error following 4b70d77b198af76ed8fe0851c3ad224074b21ed2

### ISSUE THREAD

**Issue by JCavallo** [paupino/rust-decimal#274]
```
error[E0599]: no method named `powf` found for type `f64` in the current scope
    --> /home/giovanni/.cargo/registry/src/github.com-1ecc6299db9ec823/rust_decimal-1.8.0/src/decimal.rs:2666:35
     |
2666 |             let round_to = 10_f64.powf(self.scale() as f64);
     |                                   ^^^^ method not found in `f64`
```

rustc version 1.46.0

I could compile after reverting the change locally.

**Comment by paupino** [comment]
I've just released `1.8.1` which reintroduces `std` as the default. Does this fix your issue?

**Comment by JCavallo** [comment]
@paupino It compiles successfully now, so I'm closing. Thanks !
